mainapp/templates/mainapp/request_form.html

```diff
@@ -1,75 +1,111 @@
 {% extends 'base.html' %}
 {% load bootstrap3 %}
 {% load static %}
+{% block css %}
+	<style type="text/css">
```

Comment:
CSS should be defined inside the `.css` file only!

Reply:
Adding to main styles.css caused issues since that was already cached. Adding a separate CSS file seemed unnecessary, So went with this as our primary focus is getting things done fast and quick and not on long term project maintenance and Following the best architecture. 

But I completely agree with you. Don't have access to laptop as of now, will resolve later.